### PR TITLE
[Do not merge] Fix/auth infinite loop

### PR DESCRIPTION
**Pull request by Punpun1643**
Issue: infinite loop to calling `promptAsync`

Reason:
- Every time a page render, `onPress` is immediately called. Therefore, this always initate a call to whatever function that is passed to `onPress`
- To prevent this, we need to _lazily_ call the function pass to `onPress`, this prevents the function from being executed i.e. `onPress={() => yourFunction()}`

### CODE REVIEW

File: app/app/index.js [hunk 1]
```diff
@@ -1,96 +1,156 @@
...
-          Forget Password
+        <FlatButton
+          text={'Sign In'}
+          onPress={handleLogin}
+          invert={'n'}
+          disabled={false}
+        />
+        <FlatButton
+          text={'Sign In with Google'}
+          onPress={() => promptAsync()} // you need to do "lazily" call promptAsync. Previously it was onPress={promptAsync} which means that everytime you rendered the app, it always invoke the auth session
```
Comment (Punpun1643) [inline app/app/index.js:127]: @chenxk619 @BenjoBurger see this for the fix, thanks!

Comment (Punpun1643) [reply app/app/index.js:127]: Every time you have `on...` its almost always a lazy call to some function 😄

**Comment by chenxk619** [comment]
closed to prevent other unnecessary files

File: app/app/index.js (hunk 1 above)
Comment (chenxk619) [reply app/app/index.js:127]: thank you!!

**Pull request closed without merging**